packages/components/src/horizontal-bar-list/horizontal-bar-grid-item.jsx

```diff
@@ -0,0 +1,29 @@
+// import classnames from 'classnames';
+import React from 'react';
+
+import './style.scss';
+
+const BASE_CLASS_NAME = 'horizontalbarlist';
+
+const HorizontalBarListItem = ( { data, referenceValue } ) => {
```

Comment:
Could `maxValue` be more intuitive than `referenceValue`?

Reply:
Good idea 👍